web/permissions/permissions.go

```diff
@@ -39,6 +40,8 @@ const ContextPermissionSet = "permissions_set"
 // #nosec
 const ContextClaims = "token_claims"
 
+const ShortCodeLen = 12
```

Comment:
Why do you declare two const for the short code length? One in this file and the other in pkg/sharing/sharing.go